Turkey/Program.cs

```diff
@@ -37,12 +37,20 @@ public class Program
             Arity = ArgumentArity.ZeroOrMore
         };
 
+        public static readonly Option<IEnumerable<string>> testsOption = new Option<IEnumerable<string>>(
+            new string[] { "--tests" },
+            "Run specific tests within the test root directory.")
+        {
+            Arity = ArgumentArity.OneOrMore
```

Comment:
issue(blocking): Shouldn't that be `ZeroOrMore`? Otherwise it requires to specify at least one test name.
```suggestion
            Arity = ArgumentArity.ZeroOrMore
```

Reply:
Arity determines the number of arguments for a parameter. This means that if you specify `--test-name`, you need one or more arguments. If that was `ZeroOrMore`, you'd be able to specify `--test-name` without a name, for example.

See: https://learn.microsoft.com/en-us/dotnet/standard/commandline/syntax#argument-arity

Reply:
Though I think that there's some kind of bug in System.CommandLine that for arity `n`, you need to specify individual parameters, e.g. `--test-name a --test-name b`. It won't work if you do `--test-name a b`, even though it should.

Reply:
note: Okay, thanks for the clarification. Setting this to resolved and merging.